code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -1,5 +1,7 @@
 /mob/living/carbon
 	gender = MALE
+	/// The body type associated with the body
+	var/body_type = MALE
```

Comment:
I don't really see this being used for anything but humans, is there a good reason to have it here?